packages/components-js/tests/e2e/specs/carousel.e2e.ts

```diff
@@ -229,6 +229,24 @@ test('should update pagination on next button clicks', async ({ page }) => {
   expect(await getCssClasses(bullet3)).toBe('bullet');
 });
 
+test('should update pagination in focusOnCenterSlide mode if slidesPerPage changed dynamically', async ({ page }) => {
```

Comment:
Maybe would be good to check if the pagination also still works and not only if the amount if correct?